semantic-ast/BUILD.bazel

```diff
@@ -46,3 +47,37 @@ haskell_library(
         "@stackage//:unordered-containers",
     ],
 )
+
+all_ts_deps = ["@tree-sitter-{name}//:tree-sitter-{name}".format(name = name) for name in [
+    "go",
+    "java",
+    "json",
+    "php",
+    "python",
+    "ql",
+    "ruby",
+    "rust",
+    "tsx",
+    "typescript",
+]]
```

Comment:
List comprehensions in build files are the bee’s knees. Indeed, `for` loops are prohibited at the top-level in Starlark files, so list comprehensions are our only way of iteration. Really, Starlark is kind of like a functional subset of Python.